packages/govuk-frontend/src/govuk/helpers/_typography.scss

```
@@ -114,7 +138,7 @@
 ///
 /// @access public
 
-@mixin govuk-typography-responsive($size, $override-line-height: false, $important: false) {
+@mixin govuk-font-size($size, $override-line-height: false, $important: false) {
```

Fab work in this PR 🙌

Do we align `govuk-font()` and `govuk-font-size()` and use **$line-height** in both?

```suggestion
@mixin govuk-font-size($size, $line-height: false, $important: false) {
```

---

I think yes. This is going to be released with the initial deprecation alongside changes to our docs, so it's potentially more likely that, if we release these holistically, there will be fewer instances of users changing `govuk-typography-responsive` to `govuk-font-size` and more instances of changing an incorrect use of `govuk-font` to `govuk-font-size`. Aligning these values would make that part of the update a little simpler.

---

Done. I ran into some interesting problems with doing this. We already use a var called `$line-height` in the mixin and it was being renamed on every loop which led to `_govuk-line-height` breaking:
https://github.com/alphagov/govuk-frontend/blob/f837f9c003a50f9e64a2fe3476be91e6e228f318/packages/govuk-frontend/src/govuk/helpers/_typography.scss#L128-L131
I fixed it by renaming `$line-height` to `$calculated-line-height`. Not my finest work naming wise so I'm open to alternatives.

Additionally I'm not sure if I should also deprecate this in `govuk-typography-responsive`. My instinct says no.